mobify/platform-scaffold
# Only count item as "in stock" if add to cart button exists

# Only count item as "in stock" if add to cart button exists

The previous `getAvailabilityFrom` util was failing in the case where a user would configure a product. The stock SKU on desktop was showing the product's name instead of the remaining stock. This PR checks the add to cart button instead

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1425

## Changes
- Refactor `getAvailabilityFrom` parser

## Todos

## How to test-drive this PR
- Use merlin's connector
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Ensure items that are out of stock display as out of stock on PDPs (unicorn blood)
